src/network/stk_host.cpp

```diff
@@ -214,18 +214,18 @@ void STKHost::setupServer(uint32_t address, uint16_t port, int peer_count,
                             int channel_limit, uint32_t max_incoming_bandwidth,
                             uint32_t max_outgoing_bandwidth)
 {
-    ENetAddress* addr = (ENetAddress*)(malloc(sizeof(ENetAddress)));
-    addr->host = address;
-    addr->port = port;
+    ENetAddress addr;
+    addr.host = address;
+    addr.port = port;
```

Comment:
Please do not do any changes to the networking code, it is currently being completely rewritten in a different branch, so applying this would cause a lot of unnecessary conflicts later.